cosmos/operators/_asynchronous/bigquery.py

```diff
@@ -150,7 +150,13 @@ def get_remote_sql(self) -> str:
 
         if not settings.AIRFLOW_IO_AVAILABLE:  # pragma: no cover
             raise CosmosValueError(f"Cosmos async support is only available starting in Airflow 2.8 or later.")
-        from airflow.io.path import ObjectStoragePath
+        try:
```

Comment:
Since we're checking `settings.AIRFLOW_IO_AVAILABLE` before, one of these two paths will have to work.

```suggestion
        try:
            from airflow.sdk import ObjectStoragePath
        except ImportError:
            from airflow.io.path import ObjectStoragePath
```